docs/routes/api/dev.md

```diff
@@ -8,7 +8,7 @@ active: true
 
 # solid-start dev
 
-##### `solid-start dev` starts a development server backed by [Vite](https://vitejs.dev/).
+##### `solid-start dev` starts a development server backed by [Vinxi](https://vinxi.vercel.app/).
```

Comment:
I'm a bit confused about this - all the updated examples used `vinxi dev` not `solid-start dev`. Same for `vinxi build` and `vinxi start`

Reply:
You are right. There is no solid-start commands after thus

Reply:
okay - i've made a small pr here to clarify that https://github.com/solidjs/solid-start/pull/1145